platformio/boards/arduino.json

```diff
@@ -708,5 +708,61 @@
         },
         "url": "http://arduino.cc/en/Main/arduinoBoardDue",
         "vendor": "Arduino"
+    },
+    "zero": {
+        "build": {
+            "core": "arduino_zero",
+            "extra_flags": "-DARDUINO_SAMD_ZERO -DARDUINO_ARCH_SAMD -D__SAMD21G18A__",
+            "f_cpu": "48000000L",
+            "mcu": "samd21g18a",
+            "cpu": "cortex-m0plus",
+            "pid": "0x804d",
+            "usb_product": "Arduino Zero",
+            "variant": "arduino_zero",
+            "vid": "0x2341",
+            "ldscript": "flash_with_bootloader.ld"
+        },
+        "frameworks": ["arduino"],
+        "name": "Arduino Zero (Programming Port)",
+        "platform": "atmelsam",
```

Comment:
shouldn't the platform be "atmelsamd" to differentiate with SAM3X/Due, different device architecture and match Arduino core naming?